lib/transactional.pm

```diff
@@ -165,15 +172,16 @@ sub trup_call {
     $script .= "; echo trup-\$?- | tee -a /dev/$serialdev" unless $cmd =~ /reboot / && $args{exit_code} == 0;
     script_run $script, 0;
     if ($cmd =~ /pkg |ptf /) {
-        if (wait_serial "Continue?") {
+        if ($cmd =~ /(^|\s)-\w*n\w* pkg/) {
+            record_info 'non-interactive', 'The transactional-update command is in non-interactive mode';
+        } elsif (wait_serial "Continue?") {
```

Comment:
I wish we could add a unit test for this kind of thing by extracting it.

Reply:
I don't understand :/

Reply:
> I don't understand :/

I mean, if we had the condition, in a sort routine, we could easily test the code, like we do for `version_check` for instance but we can think about this in the following weeks (I'm just making a note here)